src/util/make-knex.js

```diff
@@ -8,9 +8,8 @@ import QueryInterface from '../query/methods';
 import * as helpers from '../helpers';
 import { assign } from 'lodash'
 import BatchInsert from './batchInsert';
-import Knex from "../index";
 
-export default function makeKnex(client) {
+export default function makeKnex(client, version) {
```

Comment:
Yep... actually we already have that version in external package.json file, which was my second option for implementing this :) I'll change this one to read package json directly.